tensorflow/training/docker/2.14/py3/cu118/Dockerfile.gpu

```diff
@@ -0,0 +1,406 @@
+FROM nvidia/cuda:11.8.0-base-ubuntu20.04 AS base_image
+
+ENV DEBIAN_FRONTEND=noninteractive \
+    LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:/usr/local/lib"
+
+RUN apt-get update \
```

Comment:
nit: is it worth maintaining this in its own multistage target, as opposed to just merging this statement with the apt-get commands in the common target?

Reply:
Responded in https://github.com/aws/deep-learning-containers/pull/3643/files#r1486946235. Let's continue this thread there.